steppyngstounes/__init__.py

```diff
@@ -2,6 +2,8 @@
 from .checkpointStepper import *
```

Comment:
Are all these import *s necessary? Why not just import specific things?

Reply:
It makes it more difficult for others reading the code to figure out where things come from.

Reply:
I was trying to be general (it's what FiPy and numpy do), but it just made everything opaque. I've switched to explicit imports.